sdk/keyvault/Azure.Security.KeyVault.Certificates/MigrationGuide.md

```diff
@@ -0,0 +1,399 @@
+# Migrate from Microsoft.Azure.KeyVault to Azure.Security.KeyVault.Certificates
+
+This guide is intended to assist in the migration to version 4 of the Key Vault client library [`Azure.Security.KeyVault.Certificates`](https://www.nuget.org/packages/Azure.Security.KeyVault.Certificates) from version 3 of [`Microsoft.Azure.KeyVault`](https://www.nuget.org/packages/Microsoft.Azure.KeyVault). It will focus on side-by-side comparisons for similar operations between the two packages.
+
+Familiarity with the `Microsoft.Azure.KeyVault` library is assumed. For those new to the Key Vault client library for .NET, please refer to the [`Azure.Security.KeyVault.Certificates` README](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md) and [`Azure.Security.KeyVault.Certificates` samples](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/keyvault/Azure.Security.KeyVault.Certificates/samples) for the `Azure.Security.KeyVault.Certificates` library rather than this guide.
+
+## Table of contents
+
+- [Migration benefits](#migration-benefits)
+- [General changes](#general-changes)
+  - [Package and namespaces](#package-and-namespaces)
+  - [Separate clients](#separate-clients)
+  - [Client constructors](#client-constructors)
+  - [Creating certificate policies](#creating-certificate-policies)
+  - [Creating certificates](#creating-certificates)
+  - [Importing certificates](#importing-certificates)
+  - [Listing certificates](#listing-certificates)
+  - [Deleting certificates](#deleting-certificates)
+- [Additional samples](#additional-samples)
+
+## Migration benefits
+
+A natural question to ask when considering whether or not to adopt a new version or library is what the benefits of doing so would be. As Azure has matured and been embraced by a more diverse group of developers, we have been focused on learning the patterns and practices to best support developer productivity and to understand the gaps that the .NET client libraries have.
```

Comment:
As I was reading this, I was going to comment that I wanted to steal and reuse it.  Then I realized why I liked it so much.  😛

Reply:
:) I "stole" it from Service Bus, though maybe the text is the same. @ramya-rao-a et. al. reformatted SB from Event Hubs. We probably want to do the same with EH. The side-by-side table doesn't render well, though the intent is certainly appreciated (e.g. I like side-by-side diffs better when doing reviews).

Reply:
I agree with respect to the table, personally.   That was the format that was agreed upon across the languages for Event Hubs at the time.   I'll defer to @ramya-rao-a with respect as to whether we can either deviate per-language, coordinate cross-language on format updates, or leave as-is.